### ISSUE TITLE
On Login query a user on network/server autojoin cuts off info bar height.

### ISSUE THREAD

**Issue by un1versal** [kvirc/KVIrc#1883]
### Expected

![capture](https://cloud.githubusercontent.com/assets/3521959/13213695/9ec62324-d942-11e5-84fd-bda0c32eff18.PNG)
### Behavior

![capture1](https://cloud.githubusercontent.com/assets/3521959/13213751/f5d6710a-d942-11e5-8d04-ac6e2d46fce9.PNG)
### Steps to reproduce the issue

1) Setup a server/network setup to autojoin like so.
![capture](https://cloud.githubusercontent.com/assets/3521959/14323007/b73dbf7e-fc16-11e5-893f-31a14f20f078.PNG)

2) setup On Login tab to query a user like:
![capture](https://cloud.githubusercontent.com/assets/3521959/14322704/b5c80646-fc15-11e5-902c-337889f6d590.PNG)

3) Now with all these saved next time you start KVIrc watch what happens with above setup
### System Info

Linux Mint 17.3

```
KVIrc 4.9.1 'Aria'

Runtime Info:
System name: Linux 3.13.0-76-generic
System version: #120-Ubuntu SMP Mon Jan 18 15:59:10 UTC 2016
Architecture: x86_64
Qt version: 5.2.1
Qt theme: fusion

Build Info:
Build date: 2016-02-21 21:25:48 UTC
Sources date: 20160102
Revision number: git-6240-gaedefa9
System name: Linux-3.13.0-76-generic
CPU name: x86_64
Build command: /usr/bin/cmake
Build flags: 
   MANDIR=man
   CMAKE_INSTALL_PREFIX=/usr
   LIB_SUFFIX=/x86_64-linux-gnu
   Threads=POSIX
Compiler name: /usr/bin/c++
Compiler flags: N/A
Qt version: 5.2.1
```

**Comment by un1versal** [comment]
A workaround is to add a timer with a delay of e.g 5000ms

But tbh this isnt ideal

Can anyone confirm and maybe give some pointers?

**Comment by un1versal** [comment]
Afaik, this is no longer an issue in current nightly builds, no idea which commit fixed it though.

Closing this issue then.

**Comment by IceN9ne** [comment]
This was fixed with ed56eb2.

My apologies for not seeing that there was an open issue for this.
